### ISSUE TITLE
libav-tools is no longer available in Debian

### ISSUE THREAD

**Issue by jonorthwash** [SwatPhonLab/UltraTrace#176]
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libav-tools is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libav-tools' has no installation candidate
```

The `install.sh` script will need to be updated to take this into account.

**Comment by keggsmurph21** [comment]
Hm, we already require you to install `ffmpeg`, which itself depends on all the `av*` stuff we care about (I assume):
```
$ apt-cache depends ffmpeg
ffmpeg
 |Depends: libavcodec57
  Depends: libavcodec-extra57
  Depends: libavdevice57
 |Depends: libavfilter6
  Depends: libavfilter-extra6
  Depends: libavformat57
  Depends: libavresample3
  Depends: libavutil55
  Depends: libc6
  Depends: libpostproc54
  Depends: libsdl2-2.0-0
  Depends: libswresample2
  Depends: libswscale4
  Breaks: <libav-tools>
  Breaks: <qt-faststart>
  Suggests: ffmpeg-doc
  Replaces: <libav-tools>
  Replaces: <qt-faststart>
```
I think we can probably just remove the references to `libav-tools`.

**Comment by jonorthwash** [comment]
Huh, it even says ffmpeg replaces libav-tools.  Do you know what it is that UltraTrace needs from there?

**Comment by keggsmurph21** [comment]
Haha I don't think we even need `ffmpeg`: https://github.com/SwatPhonLab/UltraTrace/pull/181/commits/d2393e31925ee1794dc9dbdbd35ac8848c683183 seems to work ok?

**Comment by qiyoulu** [comment]
ffprobe comes with ffmpeg (on Fedora, at least) and removing ffmpeg gives this error when running `python3 -m ultratrace`

`/usr/local/lib/python3.11/site-packages/pydub-0.23.1-py3.11.egg/pydub/utils.py:193: RuntimeWarning: Couldn't find ffprobe or avprobe - defaulting to ffprobe, but may not work
  warn("Couldn't find ffprobe or avprobe - defaulting to ffprobe, but may not work", RuntimeWarning)
ERROR:  Unable to load audio file: '../ultrasound-data-example-master/20150629171639.flac' [Errno 2] No such file or directory: 'ffprobe'`

**Comment by jonorthwash** [comment]
Yeah, `ffmpeg` is a dependency of `pydub`.  What do we rely on `pydub` for again?

**Comment by jonorthwash** [comment]
> Haha I don't think we even need `ffmpeg`: [d2393e3](https://github.com/SwatPhonLab/UltraTrace/commit/d2393e31925ee1794dc9dbdbd35ac8848c683183) seems to work ok?

There's a difference between starting up successfully and actually doing things.  Do you have access to data to test with, @keggsmurph21?
